admin/spec/helpers/solidus_admin/layout_helper_spec.rb

```diff
@@ -19,9 +19,7 @@
     end
 
     it "includes the translated controller base name" do
-      expect(
-        helper.solidus_admin_title(store_name: "My Store", controller_name: "orders")
-      ).to include("Orders")
+      expect(helper.solidus_admin_title(store_name: "My Store", controller_name: 'products')).to eq("My Store - Products")
```

Comment:
While updating this spec I was wondering if we shouldn't ensure that the defaults work, i.e. checking that `current_store` (which is provided by `Spree::Core::ControllerHelpers::Store`) is available as a helper and so on. 

Also the previous expectation was passing even if the translation was missing, should the translation backend be injected as well?

My feeling is that we're somewhat fighting the system with this approach and we would be better off not trying to inject every dependency and only do that if there's a significant advantage over a more common use of instance and partial doubles:

```rb
    it "includes the translated controller base name" do
      helper.extend Spree::Core::ControllerHelpers::Store

      allow(helper).to receive(:current_store).and_return(build(:store, name: "My Store"))
      allow(helper).to receive(:controller_name).and_return("products")

      expect(helper.solidus_admin_title).to eq("My Store - Products")
    end
```

@waiting-for-dev and @solidusio/nebulab I'd be interested in knowing how you would approach it given the context above 💭

Reply:
Feel free to change it, @elia.